src/components/Counter/Counter.tsx

```diff
@@ -11,6 +11,8 @@ export interface CounterProps extends TPAComponentProps {
   incrementAriaLabel: string;
   decrementAriaLabel: string;
   inputAriaLabel: string;
+  counterAriaLabelledBy?: string;
+  counterAriaLabel?: string;
```

Comment:
So, I hope i'm not driving you crazy, but after thinking about it,
I think it would be best to rename the props to be:
```jsx
'aria-labelledby'?: string;
'aria-label'?: string;
```
This goes with the convention and best practices we're trying to implement in all our libraries now.

Reply:
shouldn't props be in camelCase? just asking.

Reply:
That's true and a legit question!
When it comes to real html attributes, we have a convention of using the real attribute names.
that means using `data-*` and `aria-*` attributes as we would have used them in regular HTML, since React supports this.

Reply:
That being said, I am pondering if having a `regionAriaLabel` and `regionAriaLabelledBy` might be a more correct prop than the `aria-` props.
What do you think?

Reply:
after looking at other components in this project I see that this convention is being used throughout, so it'll be better to leave it like this and not brake the pattern. aria-labelledby it is?

Reply:
@boriskaWix 👍